driver/pelican/interface.md

```diff
@@ -0,0 +1,40 @@
+### Pelican Schedule Interface Outline
+
+##### Context
+
+The following structs define the way users interact with the Scheduling interface of the Pelican thermostats. These structs are used to define the weekly schedule and will be interpretted by the thermSchedule.go code to retrieve/view in addition to making changes to the existing schedule for individual thermostats within a particular site. These structs are public and accessible to anyone who subscribes to the allotted endpoint or publishes to the assigned signal.
+
+##### Schedule Structs
+
+// Struct mapping each day of the week to its daily schedule <br>
+type ThermostatSchedule struct {<br>
+&nbsp;&nbsp;&nbsp;DaySchedules map[string]ThermostatDaySchedule `msgpack:"day_schedules"`<br>
+}<br>
+
+// Struct containing a series of blocks that describes a one day schedule <br>
+type ThermostatDaySchedule struct { <br>
+&nbsp;&nbsp;&nbsp;Blocks []ThermostatBlockSchedule `msgpack:blocks` <br>
+} <br>
+
+// Struct containing data defining the settings of each schedule block <br>
+type ThermostatBlockSchedule struct { <br>
+&nbsp;&nbsp;&nbsp;CoolSetting float64 `msgpack:"cool_setting"` <br>
```

Comment:
Can you document what each of the fields mean?

Reply:
Added comments describing each field!